### PR TITLE
[TSI] Split clients into separate files

### PR DESCRIPTION
**Pull request by barustum**
No code changes. Just splitting client and modify code accordingly

### CODE REVIEW

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/InstancesClient.cs [hunk 1]
```diff
@@ -0,0 +1,846 @@
...
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Time Series Insights instances client.
+    /// </summary>
+    public class InstancesClient
```
Comment (azabbasi) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/InstancesClient.cs:17]: What is the guideline on naming these? should we call them `Instances` or `InstancesClient`?

Comment (azabbasi) [reply sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/InstancesClient.cs:17]: ah, I see what you are doing there, I like it. The property is called `Instances` but its type is `InstancesClient`

**Review by azabbasi — APPROVED** [review]

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs [hunk 2]
```diff
@@ -35,6 +30,21 @@ public class TimeSeriesInsightsClient
         /// </summary>
         public virtual ModelSettingsClient ModelSettings { get; private set; }
 
+        /// <summary>
+        /// Instances client to perform various Time Series Insights instances operations.
+        /// </summary>
+        public virtual InstancesClient Instances { get; private set; }
+
+        /// <summary>
+        /// Types client to perform various Time Seris Insights types operations.
```
Comment (bikamani) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs:39]: Typo Series

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TypesClient.cs [hunk 3]
```diff
@@ -0,0 +1,663 @@
...
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Types client that can be used to perform operations for Time Series types.
```
Comment (bikamani) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TypesClient.cs:15]: I would add something like

operations to CREATE, READ, UPDATE, and DELETE on Time Series Types and their associated variables. (however you want to keep the case for create/read etc)

The same comment applies to other clients.

File: sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/QueryClient.cs [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.IoT.TimeSeriesInsights
+{
+    /// <summary>
+    /// Query client that can be used to perform query operations on Time Series Insights.
```
Comment (bikamani) [inline sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/QueryClient.cs:13]: query operations for events, series, and aggregates

**Review by bikamani — APPROVED** [review]
:shipit:

**Pull request merged**